controllers/dbaas.redhat.com/dbaasprovider_reconciler.go

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dbaasredhatcom
+
+import (
+	"context"
+	"fmt"
+	dbaasoperator "github.com/RHEcosystemAppEng/dbaas-operator/api/v1alpha1"
+	"github.com/go-logr/logr"
+
+	v1 "k8s.io/api/apps/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	label "k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/utils/pointer"
+	"os"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"time"
+)
+
+const (
+	PROVIDER           = "Red Hat DBaaS / Crunchy Bridge"
+	DISPLAYNAME        = "Crunchy Bridge managed PostgreSQL"
+	DISPLAYDESCRIPTION = " The Crunchy Bridge Fully Managed Postgres as a Service."
```

Comment:
Definitely need guidance on these values from @wilybrace and co. :)

Reply:
I confirmed displayname previously, so that seems fine to me.
I'd request display description draw from our product page:
"The managed PostgreSQL database service that allows you to focus on your application, not your database. Harness the power of Postgres on the cloud provider of your choice with trusted Crunchy Data support."

If there's a size/wrapping limit to the description text, I can work with our folks to get a desired set of language.

Reply:
we can give it a a shot and see what it looks like. Thanks!

Reply:
updated display description accordingly.